benchmark/test/reference/conversion.profile.stdout

```diff
@@ -1,28 +1,29 @@
 [
     {
-        "size": 100,
-        "stencil": "7pt",
+        "operator": {
+            "stencil": {
+                "name": "7pt",
+                "size": 100
+            }
+        },
         "conversion": {
             "coo-read": {
                 "time": 1.0,
-                "repetitions": 1,
-                "completed": true
+                "repetitions": 1
             },
             "coo-csr": {
                 "time": 1.0,
-                "repetitions": 1,
-                "completed": true
+                "repetitions": 1
             },
             "csr-read": {
                 "time": 1.0,
-                "repetitions": 1,
-                "completed": true
+                "repetitions": 1
             },
             "csr-coo": {
                 "time": 1.0,
-                "repetitions": 1,
-                "completed": true
-            }
+                "repetitions": 1
+            },
+            "completed": true
```

Comment:
For my understanding: what happens, if individual benchmarks in here throw exceptions? Is completed then false for everything?